modules/reitit-ring/src/reitit/ring.cljc

```diff
@@ -239,7 +241,7 @@
      | -------------------|-------------|
      | :parameter         | optional name of the wildcard parameter, defaults to unnamed keyword `:`
      | :root              | optional resource root, defaults to `\"public\"`
-     | :path              | optional path to mount the handler to. Works only if mounted outside of a router.
+     | :path              | path to mount the handler to. Required when mounted outside of a router, does not work inside a router.
      | :loader            | optional class loader to resolve the resources
      | :index-files       | optional vector of index-files to look in a resource directory, defaults to `[\"index.html\"]`
      | :not-found-handler | optional handler function to use if the requested resource is missing (404 Not Found)"
```

Comment:
I wonder if this is misleading (I am not sure) when it seems that a 404 Not Found may not always be returned... or will it? Applies to both documented instances of `:not-found-handler`.

Reply:
Here's how it is supposed to work: if `:not-found-handler` is set and the request is to a path that could be served by the file/resource handler (inside router, any request it gets; outside router, requests that match `:path`), but the file/resource is not found, then not-found-handler gets called.

I think this docstring is okay. Not sure how to improve it, but willing to take suggestions.

Reply:
So, do we still return a 404 Not Found for requests coming from outside the router (ie. when `:path` is defined)? If so, then omission of `(404 Not Found)` would suffice.

Reply:
If you give this parameter and set it to a handler that returns 404, then yes 😛

Reply:
Well, in any case, this is more like a [nit] and we can address this as a separate PR if it truly is a concern for anyone.